src/librustc_middle/mir/visit.rs

```diff
@@ -780,16 +780,15 @@ macro_rules! make_mir_visitor {
                 let VarDebugInfo {
                     name: _,
                     source_info,
-                    place,
+                    value,
                 } = var_debug_info;
 
                 self.visit_source_info(source_info);
                 let location = START_BLOCK.start_location();
-                self.visit_place(
-                    place,
-                    PlaceContext::NonUse(NonUseContext::VarDebugInfo),
-                    location,
-                );
+                match value {
+                    VarDebugInfoContents::Const(c) => self.visit_const(c, location),
+                    VarDebugInfoContents::Place(place) => self.visit_place(place, PlaceContext::NonUse(NonUseContext::VarDebugInfo), location),
```

Comment:
This line is 143 columns long, how does it pass `rustfmt`?

Reply:
Hmm maybe rustfmt doesn't know what to do since it's in a macro definition? I'm not familiar with rustfmt internals so I'll just wrap it manually.